Cargo.toml

```diff
@@ -5,5 +5,43 @@ members = [
     "autoendpoint",
 ]
 
+[workspace.package]
+version = "1.65.1"
+authors = [
+  "Ben Bangert <ben@groovie.org>",
+  "JR Conlin <jrconlin@mozilla.com>",
+  "Phil Jenvey <pjenvey@underboss.org>",
+  "Alex Crichton <alex@alexcrichton.com>",
+  "Mark Drobnak <mdrobnak@mozilla.com>",
+]
+edition = "2021"
+
+[workspace.dependencies]
```

Comment:
Cool, I hadn't even noticed this stuff in the 1.64 release notes. cc: @ethowitz

Reply:
This is neat!